pkg/antctl/raw/check/cluster/test_checkcontrolplaneavailability.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2024 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+import (
+	"context"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type checkControlPlaneAvailability struct{}
+
+func init() {
+	RegisterTest("Check Control Plane Availability", &checkControlPlaneAvailability{})
+}
+
+func (t *checkControlPlaneAvailability) Run(ctx context.Context, testContext *testContext) error {
+	controlPlaneLabel := "node-role.kubernetes.io/control-plane"
+	masterNodeLabel := "node-role.kubernetes.io/master"
+	controlPlaneNode, err := testContext.client.CoreV1().Nodes().List(ctx, metav1.ListOptions{LabelSelector: controlPlaneLabel})
+	if err != nil {
+		return fmt.Errorf("failed to list control plane Nodes: %w", err)
+	}
+	masterNode, err := testContext.client.CoreV1().Nodes().List(ctx, metav1.ListOptions{LabelSelector: masterNodeLabel})
```

Comment:
we avoid that term ("master") in the codebase whenever possible. See https://github.com/antrea-io/antrea/blob/main/CONTRIBUTING.md#inclusive-naming. Now we cannot avoid it when we have to spell out the actual K8s label name, but we can avoid it in variable names:
```go
controlPlaneNodes := sets.New[string]()
controlPlaneLabels := []string{"node-role.kubernetes.io/control-plane", "node-role.kubernetes.io/master"}
for _, label := range controlPlaneLabels {
        nodes, err := testContext.client.CoreV1().Nodes().List(ctx, metav1.ListOptions{LabelSelector: label})
	if err != nil {
		return fmt.Errorf("failed to list Nodes with label %s: %w", label, err)
	}
	for idx := range nodes.Items {
		controlPlaneNodes.Insert(nodes.Items[idx].Name)
	}
}

if controlPlaneNodes.Empty() {
	// ...
}
// ...
```

Reply:
Thanks for the information :)